### DISCUSSION TITLE
微信小程序中按文档引入后，ts类型检测异常，导致运行不起来

### DISCUSSION THREAD

**Discussion by tiantaoTT** [Tencent/libpag#1708] [General] [answered]
![企业微信截图_1693818965596](https://github.com/Tencent/libpag/assets/48501683/862df6eb-55f3-44e7-ab51-67eb2226a411)
如图这样，install最新版本4.2.100，这是还缺少什么依赖吗？

**Comment by tiantaoTT** [comment]
缺少webgl2依赖包？

**Comment by zhalice2011** [comment, marked as answer]
你好, 我按照文档操作, 安装 4.2.100 版本, 可以正常运行示例demo

文档: https://github.com/Tencent/libpag/tree/main/web/wechat

<img width="716" alt="image" src="https://github.com/Tencent/libpag/assets/24289349/5b61cc5e-0246-430f-966a-57c572b7cee1">

![wecom-temp-a92973d79fb1b4a2e41c512e71996354](https://github.com/Tencent/libpag/assets/24289349/f6fdf244-c62f-4300-a054-6390abdf6362)

如果还是遇到问题试试安装 @types/emscripten看看, 或者提供更详细的信息
